subprojects/plugins/src/integTest/groovy/org/gradle/api/plugins/JvmTestSuitePluginIntegrationTest.groovy

```diff
@@ -359,18 +359,20 @@ Artifacts
                 visible = false
                 canBeResolved = true
                 canBeConsumed = false
-                extendsFrom(configurations.implementation)
-                attributes {
-                    attribute(Category.CATEGORY_ATTRIBUTE, objects.named(Category, Category.VERIFICATION))
-                    attribute(VerificationType.VERIFICATION_TYPE_ATTRIBUTE, objects.named(VerificationType, VerificationType.TEST_RESULTS))
-                }
             }
+            testDataConfig.dependencies.add(project.dependencies.create(project))
```

Comment:
Looking at what a few of these tests are actually doing.... this is probably an example of over testing. I think at the time, we didn't have the aggregation plugins written yet, so we needed to test this separately.

I would...
- Remove the tests that create configurations and artifact views (the "via Dependency Management" tests). This behavior is covered in the aggregation plugin's tests.
- Keep the test that checks we're exposing the correct variant
- Keep the tests that check for duplicate test suite types